static/js/compose.vue

```diff
@@ -170,7 +170,8 @@
             constraints: {
               max_airmass: 1.6,
               min_lunar_distance: 30.0
-            }
+            },
+            completion_threshold: 90.0
```

Comment:
Can you move this up to the top where all of the other non object attributes are?

Reply:
There aren't any other non-object attributes for the request, but I moved it to the top of the request section